nfc/source/main.c

```diff
@@ -194,7 +244,7 @@ int main(int argc, char* argv[])
     consoleUpdate(NULL);
 
     // Initialize the nfp:* service.
-    rc = nfpInitialize(NfpServiceType_User);
+    rc = nfpInitialize(NfpServiceType_Debug);

```

Comment:
` // Use the NfpServiceType as required by your app, only use Debug if actually needed.`

Reply:
nfpGetAdminInfo is only available on the debug interface. I could remove it if we want to keep the user interface and hardcode the application area id instead

Reply:
Can keep it, debug-usage needs a comment however.